src/legacy/ui/ui_render/bootstrap/startup.js.hbs

```diff
@@ -0,0 +1,106 @@
+var uiSettingsJSON = window.localStorage.getItem('uiSettings') || '{}';
+
+var uiSettings = {};
+try {
+  uiSettings = JSON.parse(uiSettingsJSON) || {};
+} catch (error) {
+  console.log(error);
+}
+
+var useBrowserColorScheme = window.localStorage.getItem('useBrowserColorScheme') === 'true' || false;
+
+var browserDarkMode = uiSettings['theme:darkMode'] || {};
+var browserThemeVersion = uiSettings['theme:version'] || {};
+
+var rawDarkMode = typeof browserDarkMode.userValue !== 'boolean' ? {{configDarkMode}} : browserDarkMode.userValue;
+
+debugger;
```

Comment:
Was this left in intentionally?

Reply:
nope 🤦 - will remove.

Reply:
will you have time for this? @joshuarrrr